sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,39 @@
+function fetchAsyncSignals(mode, customFunction, customKey) {
+	console.log("Going to fetch signals");
+	var eids = "";
+	switch (mode) {
+		case 1:
+			if (pbjs && pbjs.getUserIdsAsEids && typeof pbjs.getUserIdsAsEids === "function") {
+				eids = pbjs.getUserIdsAsEids(); // Get Identities from Prebid API in oRTB eids structure
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 3 tabs but found 12 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)